tests/src/Simple/HelloWasm/HelloWasm.csproj

```diff
@@ -15,4 +15,8 @@
   </PropertyGroup>
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), SimpleTest.targets))\SimpleTest.targets" />
+
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <DefineConstants>DEBUG</DefineConstants>
```

Comment:
This will remove all the other defined constants. I find it odd that DEBUG wouldn't be defined. Can you run msbuild with `/v:diag` to see if something is removing it?

Reply:
Removed.  I must have been getting confused somewhere.  Thanks for reviewing